cmd/kpod/spec.go

```diff
@@ -110,8 +111,24 @@ func createConfigToOCISpec(config *createConfig) (*spec.Spec, error) {
 		configSpec.Linux.Seccomp = &seccompConfig
 	}
 
+	// HANDLE CAPABILITIES
+	var err error
+	var caplist []string
+	if config.privileged {
+		caplist = caps.GetAllCapabilities()
+	} else {
+		caplist, err = caps.TweakCapabilities(defaultCapabilities(), config.capAdd, config.capDrop)
+		if err != nil {
+			return nil, err
+		}
+	}
+
+	configSpec.Process.Capabilities.Bounding = caplist
+	configSpec.Process.Capabilities.Permitted = caplist
+	configSpec.Process.Capabilities.Inheritable = caplist
+	configSpec.Process.Capabilities.Effective = caplist
```

Comment:
Would you consider plopping this into a function that returned a spec.Process.Capablilties type?  I did that for some of the larger things like volumes and keeps this clean

Reply:
oh, and if you do, some unittests would be really handy.  you could come back around this, because I have a PR that initiates the tests.

Reply:
Fixed